Makefile

```diff
@@ -53,9 +53,12 @@ install:
 
 	cp -Ra dbicdh "$(DESTDIR)"/usr/share/openqa/dbicdh
 
-test:
-	OPENQA_CONFIG= prove -r
-	./script/tidy --check
+
+.PHONY: check_style
+check_style:
```

Comment:
not uncommon to me but I don't mind changing. I think guessing what suits is not a good idea :-) I am open for suggestions